### PR TITLE
Update client.json with go-redis/redis client

### PR DESCRIPTION
**Pull request by eatonphil**
I'm surprised this package isn't already in the list, but I didn't see it. It is fairly popular.

### CODE REVIEW

**Comment by cheprasov** [comment]
I think, it will be better to ask authors of client to add client here by theyself, maybe, they have some reasons not to do it yet.

**Comment by badboy** [comment]
/cc @vmihailenco

@eatonphil: JSON is broken

File: clients.json [hunk 1]
```diff
@@ -90,6 +90,15 @@
   },
 
   {
+    "name": "go-redis/redis",
+    "language": "Go",
+    "repository": "https://github.com/go-redis/redis"
+    "description": "Redis client for Golang.",
```
Comment (vmihailenco) [inline clients.json]: Probably mention that it supports Redis Sentinel and Redis Cluster out of the box.

**Comment by vmihailenco** [comment]
Thanks for adding client to the list! I don't have any objections :)

**Comment by eatonphil** [comment]
@badboy thanks, made those changes. @vmihailenco updated to include that info!

**Pull request merged**